README.md

```diff
@@ -1195,8 +1303,34 @@ async def call_tool(name: str, arguments: dict[str, Any]) -> dict[str, Any]:
             return structured
         except Exception as e:
             raise ValueError(f"Calculation error: {str(e)}")
```

Comment:
Out of scope, but this shouldn't be necessary, since the exception is just bubbled up?